.circleci/config.yml

```diff
@@ -15,6 +15,16 @@ workflows:
           filters: *all_commits
       - protobufjs-load-test:
           filters: *all_commits
+      - regenerate:
+          requires:
+            - build
```

Comment:
I feel like these should be run after `regenerate`. It's already happened where we've generated the CLI and it was broken. I'm not sure that running all of these tests prior to regeneration everything would help a lot. Unless I'm missing something?

Reply:
Agreed. Removed those requires.